flask/app/tasks.py

```diff
@@ -1,50 +1,34 @@
 import os
 from datetime import datetime, timedelta
+from itertools import chain
+
 from . import models
 from .email import send_email
 from .extensions import db
-from sqlalchemy import text
 
 
 def send_email_notification(app):
-    app.logger.info("hello")
     with app.app_context():
+        yesterday = (datetime.now() - timedelta(1)).strftime("%Y-%m-%d")
 
-        app.logger.info("Sending email notification...")
-
-        yesterday = (datetime.now() - timedelta(5)).strftime("%Y-%m-%d")
-
-        analyses = db.session.query(models.Analysis).filter(
-            models.Analysis.requested >= yesterday,
+        datasets = (
+            db.session.query(models.Dataset)
+            .join(models.Dataset.analyses)
+            .filter(models.Analysis.requested >= yesterday)
+            .all()
         )
 
-        app.logger.info("Available analyses...")
-        app.logger.debug(analyses)
+        analyses = list(chain(*[d.analyses for d in datasets]))
 
-        dynamic_object = {"analyses": []}
+        email_object = {"analyses": []}
 
         for analysis in analyses:
-            mapped_dataset_ids_gen = (
-                db.session.query(models.datasets_analyses_table)
-                .filter(
-                    models.datasets_analyses_table.c.analysis_id == analysis.analysis_id
-                )
-                .values(text("dataset_id"))
-            )
-
-            mapped_dataset_ids = [x for x, in mapped_dataset_ids_gen]
-
-            datasets = (
-                db.session.query(models.Dataset)
-                .filter(models.Dataset.dataset_id.in_(mapped_dataset_ids))
-                .all()
-            )
-
-            dynamic_object["analyses"].append(
+            email_object["analyses"].append(
                 {
+                    "analysis_id": analysis.analysis_id,
                     "requested": analysis.requested.strftime("%Y-%m-%d"),
                     "requester": analysis.requester.username,
-                    "pipeline": analysis.pipeline.pipeline_name,
+                    "pipeline": analysis.kind,
```

Comment:
Oh you know, this might also be the part that broke it. I must have missed this.